src/lib/CreateFormMixin.js

```diff
@@ -101,18 +101,18 @@ module.exports = {
   },
 
   isGroupValid: function (groupName) {
-    var isValid = true;
+    var isValid = false;
     var fields = Object.keys(this.__schema).filter(key => this.__schema[key].group === groupName);
     fields.forEach(key => {
-      if (this.validateField(key)) isValid = false;
+      if (this.validateField(key)) isValid = true;
     });
     return isValid;
   },
 
   isValid: function () {
```

Comment:
Since this function is  called `isValid`, it should return `true` if all fields are valid, and `false` all fields are not valid.  Given that, what do you need to change here?